src/components.d.ts

```diff
@@ -473,6 +473,37 @@ export namespace Components {
          */
         "value"?: string;
     }
+    interface FwToast {
+        /**
+          * The Content of the action link
+         */
+        "actionLinkText": string;
+        /**
+          * The content to be diaplyed in toast
+         */
+        "content": string;
+        /**
+          * Pause the toast from hiding on mouse hover
+         */
+        "pauseOnHover": boolean;
+        /**
+          * Pause the toast from hiding on mouse hover
+         */
+        "position": "top-center" | "top-left" | "top-right";
+        /**
+          * won't close automatically
+         */
+        "sticky": boolean;
+        /**
+          * Time duration of the toast visibility
+         */
+        "timeout": number;
+        "trigger": (configs: object) => Promise<void>;
+        /**
+          * Type of the toast - success,failure, warning, inprogress
+         */
+        "type": "success" | "error" | "warning" | "inprogress";
```

Comment:
```suggestion
        "type": "success" | "failure" | "warning" | "inprogress";
```
Just a suggestion 😄

Reply:
This not just for failure alone but can be used for any errors.